pe/write.go

```diff
@@ -80,6 +80,13 @@ func (peFile *File) Bytes() ([]byte, error) {
 			NumberOfLineNumbers:  section.NumberOfLineNumbers,
 			Characteristics:      section.Characteristics,
 		}
+
+		// if the PE file was pulled from memory, the symbol table offset in the header will be wrong.
+		// Fix it up by picking the section that lines up, and use the raw offset instead.
+		if peFile.FileHeader.PointerToSymbolTable == sectionHeader.VirtualAddress {
```

Comment:
The alternative way of doing this is having a flag that is set on the `pefile` object that indicates it was loaded from memory. Then we can _always_ do the fixup on in-memory, and _never_ otherwise.